tenant_schemas/middleware.py

```diff
@@ -75,22 +73,28 @@ class SuspiciousTenantMiddleware(TenantMiddleware):
     TENANT_NOT_FOUND_EXCEPTION = DisallowedHost
 
 
-class DefaultSchemaTenantMiddleware(TenantMiddleware):
+class DefaultTenantMiddleware(SuspiciousTenantMiddleware):
     """
-    This middleware makes it possible for you to set a default schema to be
-    selected when no schema could be determined from the hostname.
+    Extend the SuspiciousTenantMiddleware in scenario where you want to
+    configure a tenant to be served if the hostname does not match any of the
+    existing tenants.
+
+    Subclass and override DEFAULT_SCHEMA_NAME to use a schema other than the
+    public schema.
+
+        class MyTenantMiddleware(DefaultTenantMiddleware):
+            DEFAULT_SCHEMA_NAME = 'default'
     """
+    DEFAULT_SCHEMA_NAME = None
+
+    def get_tenant(self, model, hostname, request):
+        if self.DEFAULT_SCHEMA_NAME is None:
```

Comment:
Just as a matter of opinion;

```
schema_name = self.DEFAULT_SCHEMA_NAME
if not schema_name:
    schema_name = get_public_schema_name()
```

seems cleaner to me. As stated, I'm not married to this syntax. Purely opinion. What's your take?